Source/LogMessageWriter.swift

```diff
@@ -37,10 +37,27 @@ public protocol LogMessageWriter {
 /// The ConsoleWriter class runs all modifiers in the order they were created and prints the resulting message
 /// to the console.
 open class ConsoleWriter: LogMessageWriter {
+    /// Used to define whether to use the print or NSLog functions when logging to the console.
+    ///
+    /// During development, it is recommended to use the `.print` case. When deploying to production, the `.nslog`
+    /// case should be used.
+    ///
+    /// - print: The Swift `print` function that is faster, not thread safe and doesn't write to the device console.
```

Comment:
This would imply that if you have multiple loggers not all using the same queue, the default of `.print` may lead to crashes?